akka-persistence-typed/src/test/scala/akka/persistence/typed/scaladsl/PersistentBehaviorSpec.scala

```diff
@@ -148,6 +150,10 @@ object PersistentBehaviorSpec {
         case IncrementWithPersistAll(n) ⇒
           Effect.persist((0 until n).map(_ ⇒ Incremented(1)))
 
+        case cmd: IncrementWithConfirmation ⇒
+          Effect.persist(Incremented(1))
+            .thenReply(cmd)(newState ⇒ Done)
```

Comment:
I'm a :-1: to creating an alias for sending a message

Reply:
Yes,why not just a method named `ackDone`